hackr/crypto.py

```diff
@@ -0,0 +1,87 @@
+import hashlib
+import os
+import bcrypt
+import whirlpool
+
+
+class Encrypt:
+
+    def __init__(self, plain_text):
+        self.plain_text = plain_text
+
+    def sha1(self):
+        encrypted = hashlib.sha1(self.plain_text).hexdigest()
+        print encrypted
+
+    def md5(self):
+        encrypted = hashlib.md5(self.plain_text).hexdigest()
+        print encrypted
```

Comment:
`return` instead of that?
Same for other functions as well.

Reply:
sure, thanks for the correction :tada: